src/Misc/layoutbin/systemd.svc.sh.template

```diff
@@ -86,6 +84,15 @@ function install()
     chown ${run_as_uid}:${run_as_gid} ./runsvc.sh || failed "failed to set owner for runsvc.sh"
     chmod 755 ./runsvc.sh || failed "failed to set permission for runsvc.sh"
 
+    if [[ "${selinuxEnabled}" == "Enforcing" ]]; then
+        # We will need to add the initrc_exec_t context for the script
+        # (if we can't add, softfail first and try modify instead)
+        semanage fcontext --add    --type initrc_exec_t "${RUNNER_ROOT}/runsvc.sh" ||\
```

Comment:
should we error if we can't find `semanage` installed?

Reply:
Yes, we can either:
  - Bail if `selinux` is enabled and Enforcing but `semanage` isn't present; or
  - Report the issue to the user but continue to fail (as it does now).

Additionally, we could optionally check the context in case they've done something clever and the file has inherited the correct context anyway. In my tests with CentOS `semanage` was installed but right now we have more reports of it missing than present so it may have been how I did my installation.

Reply:
Well, you could probably still adjust the context even if SELinux is not enforcing. I'd just check if `semanage` is installed and try do do the context change.